deltachat-ios/Controller/Settings/Proxy/ProxySettingsViewController.swift

```diff
@@ -262,6 +277,33 @@ extension ProxySettingsViewController {
         tableView.deselectRow(at: indexPath, animated: true)
     }
 
+    override func tableView(_ tableView: UITableView, leadingSwipeActionsConfigurationForRowAt indexPath: IndexPath) -> UISwipeActionsConfiguration? {
+        guard
+            proxies.isEmpty == false,
+            indexPath.section == ProxySettingsSection.proxies.rawValue
+        else { return nil }
+
+        let shareAction = UIContextualAction(style: .destructive, title: nil) { [weak self] _, _, completion in
+            DispatchQueue.main.async {
+                self?.shareProxy(at: indexPath)
+                completion(true)
+            }
+        }
+        shareAction.backgroundColor = .systemGreen
+        shareAction.accessibilityLabel = String.localized("proxy_share_link")
+        if #available(iOS 13.0, *) {
+            shareAction.image = Utils.makeImageWithText(image: UIImage(systemName: "square.and.arrow.up"), text: String.localized("proxy_share_link"))
+        } else {
+            shareAction.title = String.localized("proxy_share_link")
```

Comment:
context is clear, we can get shorter:

```suggestion
            shareAction.image = Utils.makeImageWithText(image: UIImage(systemName: "square.and.arrow.up"), text: String.localized("menu_share"))
        } else {
            shareAction.title = String.localized("menu_share")
```